### ISSUE TITLE
Amazon app store will no longer publish stock AnkiDroid

### ISSUE THREAD

**Issue by mikehardy** [ankidroid/Anki-Android#5051]
###### Reproduction Steps

1. Attempt to upload new version of AnkiDroid to Amazon app store
2. wait for status
3. 

###### Expected Result

App is published

###### Actual Result

Boilerplate rejection "Q: Why didn’t my launcher, widget, or lockscreen app go live for Amazon devices?
Amazon seeks to preserve a consistent customer experience on Amazon devices. If your app overrides the native user experience, it will not be published on Amazon devices."
https://developer.amazon.com/docs/app-submission/viewing-app-submission-status.html

I'm assuming they can't handle the widget we include anymore even though it is optional? And hasn't been a problem before?

@nicolas-raoul sent the info and someone has the credentials. I asked Tim if he'd take a PR that did a missing-widget build for Amazon, we'll see

**Comment by mikehardy** [comment]
From Tim via forum:
```
Ah, Amazon being annoying as usual... It doesn't say anything specifically about the widget, so it could just as well be the template editor crash getting detected by their robots, so maybe let's try again with 2.8.5 before flushing out the widget?
```

So we'll wait till 2.8.5 and see

**Comment by mikehardy** [comment]
2.8.5 wasn't able to launch because of an unrelated issue (Google Play Store won't accept apps unless they are minSdk 15 or greater) so I guess this is waiting until 2.9? Unfortunately we don't have a date for that. It seems like 2.8.4 isn't available at all on the amazon app store though according to a user report https://groups.google.com/d/msgid/anki-android/529957077.9178901.1551480739507%40mail.yahoo.com however it seems to be available? https://www.amazon.com/gp/product/B004L1G7T2

**Comment by mikehardy** [comment]
I'll try this again with 2.10

**Comment by david-allison-1** [comment]
@mikehardy Any feedback on this? I have a feeling that `com.ichi2.libanki.hooks.ExternalHookLoader` will cause problems with automated tooling.

**Comment by mikehardy** [comment]
I did a quick search and was unable to find the credentials as of yet. I think @nicolas-raoul has them so I guess I have to beg you again Nicolas :-) - if you could pass over the Amazon store credentials I'd appreciate it, I know I had it at one point?!

Those hooks were in for 2.8 and they carried 2.8, so that seems odd to me, but :shrug:

**Comment by david-allison-1** [comment]
Closed in #6339
